client/src/document/ingredients/browser-compatibility-table/feature-row.tsx

```diff
@@ -127,8 +127,8 @@ const CellText = React.memo(
   }) => {
     const currentSupport = getFirst(support);
 
-    const added = currentSupport && currentSupport.version_added;
-    const removed = currentSupport && currentSupport.version_removed;
+    const added = currentSupport?.version_added ?? null;
+    const removed = currentSupport?.version_removed ?? null;
```

Comment:
Not sold but, it feels I little more readable to me to have:

```suggestion
    const added = currentSupport?.version_added != null;
    const removed = currentSupport?.version_removed != null;
```

Using `!=` as opposed to `!===` will work for both `null` and `undefined`. Just in case something changes in the future. Thoughts?

Reply:
I don't think your suggested change would work, because it effectively makes `added` and `removed` a `boolean`:

```ts
// string | boolean | null | undefined
typeof (currentSupport && currentSupport.version_added)

// string | boolean | null
currentSupport?.version_added ?? null;

// boolean
currentSupport?.version_added != null
```

Reply:
However:

```
// undefined
undefined && undefined.version_added
undefined?.version_added

// null
null && null.version_added
null?.version_added
```

So instead of changing the `added` and `removed` assignments, we could also change the `currentSupport` assignment:

```patch
-    const currentSupport = getFirst(support);
+   const currentSupport = getFirst(support) ?? null;
``

Reply:
Oh, I remember this didn't work, because some consumer of `currentSupport` expects `undefined`, but not `null`.

Reply:
Ah bugger. OK, this is all good then. Thanks, @caugner